### PR TITLE
Document reimbursements OCR dataset

### PR DESCRIPTION
**Pull request by fgrehm**
I haven't had a chance to put up the PR on serenata-toolbox with the lib for OCRing receipts but I'm opening this for some early feedback. Also, here's a new link to the csv with receipts texts: https://we.tl/iyUDWuqiBs

cc @cuducos

### CODE REVIEW

**Comment by cuducos** [comment]
As soon as [this](https://github.com/datasciencebr/serenata-toolbox/pull/40) is merged I can merge this PR ; )

**Comment by fgrehm** [comment]
@cuducos thanks! unfortunately I haven't had a chance to do send a PR to toolbox that introduces the `ReimbursementOCR` I described here. Maybe we should flag that section as WIP (or just remove) before merging?

**Comment by cuducos** [comment]
I've uploaded `2017-02-15-receipts-texts.xz` and `2017-02-15-receipts-ocr-raw.tar.xz` was already there. With `from serenata_toolbox.datasets import fetch` one can already download these file. Does it worth it to add both to `fetch_latest_backup`? What else am I missing?

**Comment by fgrehm** [comment]
[Here](https://github.com/fgrehm/serenata-de-amor/blob/2d52185906b5824147459e5e3959f7164c3a2448/docs/receipts-ocr.md#how-can-i-ocr-other-receipts) I'm talking about some code I'd like to add to the toolbox but haven't had a chance to implement yet 😄

**Comment by cuducos** [comment]
Ow… got it! I was so excited I skip that ; ) So I put this on hold until we have `ReceiptsOCR`, is that ok?

**Comment by jtemporal** [comment]
hello, @cuducos and @fgrehm [this](https://github.com/datasciencebr/serenata-toolbox/pull/40) was merged! ;) what we need to get this one merged too?

**Comment by cuducos** [comment]
AFAIK @fgrehm documented an object that isn't coded yet ([gotta love API driven development](https://www.youtube.com/watch?v=ccqGOODoQTg)): `ReimbursementsOCR`. I think that's it ; )

File: docs/receipts-ocr.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+- Some scans have really low quality and / or the receipt gets a bit "faded
+  away" due to the thermal paper many places emit, making it harder for the
+  computer to guess what's the text in it.
+
+In other words, there is room for (non trivial) improvements on this OCR process
+(like some image preprocessing) and there are things we can do ourselves in
+order to increase the chances of finding suspicious reimbursements with the PDFs
+provided.
+
+## How can I OCR other receipts?
```
Comment (fgrehm) [inline docs/receipts-ocr.md]: @cuducos @jtemporal Hey guys, in the spirit of getting this out the door for more people to use... What do you guys think about removing this section from the docs and merge just the documentation on the dataset as it is?

This should not only make this data more visible to other contributors but also allow 3 or 4 other PRs I got open with new notebooks to be merged. I'm still interested in making the code I outlined here work, but other people can benefit from this information and try to come up with new insights while I find the time to work on that.

Comment (cuducos) [inline docs/receipts-ocr.md]: Sounds like a good idea ; )

Comment (fgrehm) [inline docs/receipts-ocr.md]: Done!

**Comment by jtemporal** [comment]
I removed the WIP label, and I think this looks good to merge!

Any other considerations before merging it? cc @cuducos @fgrehm

**Comment by fgrehm** [comment]
IMO it's just :shipit: 😃

**Comment by jtemporal** [comment]
Taking into account our pattern of PR acceptance so far, it is better to wait until #254 is done to merge this ;)

File: docs/receipts-ocr.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+```python
+# Read Google's API key from the `config.ini` file
+import configparser
+settings = configparser.RawConfigParser()
+settings.read('../config.ini')
+google_api_key = settings.get('Google', 'APIKey')
+
+# Configure the client + some dirs for temporary data
+ocr = ReimbursementsOCR('../data/receipts-pdfs', '../data/receipts-imgs')
```
Comment (Irio) [inline docs/receipts-ocr.md]: Can you add a line importing this class? (I noticed it's already mentioned in the line 154).

File: docs/receipts-ocr.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+# Grab the reimbursements you're interested on
+reimbursements = pd.read_csv('../data/2016-12-06-reimbursements.xz', low_memory=False)
+reimbursements = reimbursements.query('some criteria you are interested in')
```
Comment (Irio) [inline docs/receipts-ocr.md]: Maybe change this "some criteria you are interested in" to just a comment? I'm afraid that people will try to run this whole code block and get an exception.

**Comment by Irio** [comment]
Oh it seems I missed the last commit.

**Pull request merged**